docs/src/main/sphinx/functions/list.rst

```diff
@@ -379,6 +380,7 @@ S
 
 - :func:`second`
 - :func:`sequence`
+- :ref:`sequence <sequence_table_function>`
```

Comment:
So how would a reader distinguish the two sequence functions by looking at this list .. dont we need to have some sort of differentiation @findepi and @colebow ?

Reply:
I feel like the best fix for this is renaming one of the functions - having two different functions seems problematic in general.

Reply:
Scalar and table functions are in different namespaces, so we shouldn’t rename them just because they get mixed up in this list.

An alternative would be to have a separate list for table functions, or add a disambiguation parenthetical.

Reply:
Thats what I suggested in another comment .. and I think this is better than having the same wording twice with different links

`sequence (table function)`

and only add that for table functions.. otherwise we end up with lots of `(scalar function)` 

or we could do

sequence (scalar function, table function)

and link the ones in brackets.. but personally I think that is worse than having two lines..